src/pages/Lending/hooks/useLendingQuoteQuery.ts

```diff
@@ -212,7 +211,7 @@ export const useLendingQuoteOpenQuery = ({
     if (!wallet || !_borrowAccountId || !destinationAccountMetadata || !borrowAsset) return
 
     const deviceId = await wallet.getDeviceID()
-    const pubKey = isLedger(wallet) ? fromAccountId(_borrowAccountId).account : undefined
```

Comment:
revert, this is for `getAddress` which we only want to pass through for ledger.

Reply:
One too deep indeed!  7e2afce302